bench/wtperf/wtperf.c

```diff
@@ -131,6 +146,173 @@ randomize_value(CONFIG_THREAD *thread, char *value_buf)
 }
 
 static int
+setup_truncate(
+    CONFIG *cfg, CONFIG_THREAD *thread,
+    TRUNCATE_CONFIG *trunc, WT_CURSOR *cursor) {
```

Comment:
The line wrapping doesn't look quite right to me here. Could it fit more like the ```start_threads``` method?